crates/turbopack-ecmascript/src/parse.rs

```diff
@@ -57,7 +57,9 @@ pub enum ParseResult {
         #[turbo_tasks(debug_ignore, trace_ignore)]
         source_map: Arc<swc_core::common::SourceMap>,
     },
-    Unparseable,
+    Unparseable {
+        messages: Option<Vec<String>>,
```

Comment:
```suggestion
        messages: Vec<String>,
```

Reply:
A plain string feels a bit loosy. Could that be a `Vc<Issue>`?

Reply:
Main purpose for this is to include in runtime code to throw - https://github.com/vercel/turbo/pull/7505/files#diff-40046925dbca8e1d654bd7bf2fb30362fd19391f9edd943fe019654e672833eeR872-R873 

which expects string already, not sure if it's worth string -> issue -> string again.